tests/app/forms/validation/test_mobile_number_validation.py

```diff
@@ -59,6 +60,18 @@ def test_non_numeric_string_invalid(self):
 
         self.assertEqual(error_messages["INVALID_MOBILE_NUMBER"], str(ite.exception))
 
+    def test_number_not_starting_with_7_raises_validation_error(self):
```

Comment:
Can we align the name with the others i.e _invalid rather than raises_validation_error, I also think it needs to highlight that its after the 0 gets stripped it needs to be 7

Reply:
Just checked its called the prefix service type

Reply:
so something like test_prefix_service_type_invalid, you could always write a comment that says it should be 07

Reply:
ok, renamed to test_prefix_service_type_invalid, thanks!